cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -623,6 +660,85 @@ func CmdMigrateHashRun(*cobra.Command, []string) error {
 	return migrate.WriteSumFile(dir, sum)
 }
 
+// CmdMigrateImportRun is the command executed when running the CLI with 'migrate import' args.
+func CmdMigrateImportRun(cmd *cobra.Command, _ []string) error {
+	if MigrateFlags.DirFormat == formatAtlas {
+		return fmt.Errorf("cannot import a migration directory already in %q format", formatAtlas)
+	}
+	// Create migration directories.
```

Comment:
Feels unnecessary and not correct. `dir(false)` won't create a dir.

Reply:
Comment is misleading. Create means instantiate in this case. Comment itself is a little useless.